### PR TITLE
ENH: Add default mypy and pyright tasks

### PR DESCRIPTION
**Pull request by bashtage**
Add simpler task names to simplify running tools

### CODE REVIEW

**Comment by bashtage** [comment]
@twoertwein Not sure how you feel about these, but I like the idea of just running `poe mypy` like I run `poe pytest`.  This only runs one of the mypy tasks, but that it usually enough to find any errors IME.

**Comment by twoertwein** [comment]
> @twoertwein Not sure how you feel about these, but I like the idea of just running `poe mypy` like I run `poe pytest`. This only runs one of the mypy tasks, but that it usually enough to find any errors IME.

Cannot you already do that with `poe pyright_src` and `poe mypy_src` (or did I accidentally remove that)?

**Comment by twoertwein** [comment]
Happy to shorten `mypy_src` to `mypy` and `pyright_src` to `pyright` - but we shoud not have both `mypy` and `mypy_src`

**Comment by bashtage** [comment]
> Happy to shorten `mypy_src` to `mypy` and `pyright_src` to `pyright` - but we shoud not have both `mypy` and `mypy_src`

That is my idea.  To have a "default" mypy which would just be called mypy.  Same for pyright.  I'm used to using `mypy <project>`, so something like `poe mypy` feels intuitive (like `poe pytest` since `pytest <project>` is more common).

**Comment by bashtage** [comment]
I have removed the three comands that end in `src` so that their "base"s are now the preferred names

```
poe mypy
poe pyright
poe test
```

**Review by twoertwein — APPROVED** [review]
Thank you, looks good to me!

@Dr-Irv

File: docs/tests.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ## Test
 
-- Run local tests against source code. </br> `poe test_src` </br> </br> 
+- Run local tests against source code. </br> `poe test` </br> </br>
```
Comment (Dr-Irv) [inline docs/tests.md:3]: We should say which is the default profile

**Review by Dr-Irv — CHANGES_REQUESTED** [review]
one small change in the docs

File: docs/tests.md (hunk 1 above)
Comment (twoertwein) [reply docs/tests.md:3]: There are no more profiles, that's why I opened #162

**Review by Dr-Irv — APPROVED** [review]
Thanks @bashtage

**Pull request merged**